pid_controller/src/pid_controller.yaml

```
@@ -0,0 +1,87 @@
+pid_controller:
+  dof_names: {
+    type: string_array,
+    default_value: [],
+    description: "Specifies dof_names or axes used by the controller. If 'state_dof_names' parameter is defined, then only command dof names are defined with this parameter.",
```

there is no `state_dof_names` parameter?

---

there is below `reference_and_state_dof_names`. I chose this name because both references and states have to have shared names to end up in the same order to make sensible calculation of control error.

---

but there is no parameter called `state_dof_names` but only `reference_and_state_dof_names`? or I just don't get your point :D

---

```suggestion
    description: "Specifies dof_names or axes used by the controller. If 'reference_and_state_dof_names' parameter is defined, then only command dof names are defined with this parameter.",
```
I still think that there is a typo